<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceConsumptionPage</class>
 <widget class="QWidget" name="MaintainceConsumptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}
QLabel
{
font:24px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
border:2px solid white;
}
QPushButton
{
	border-radius:8px;
font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 128, 128);border-top: 2px solid white;border-left: 2px solid white;
}</string>
  </property>
  <widget class="QPushButton" name="VVVFClearBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>牵引计数器清除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SIVClearBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>辅助计数器清除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>530</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>520</y>
     <width>1018</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);

	border:1px solid white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>起始时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>71</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>VVVF
/kwh</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>牵引</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>电阻制动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>ACM/kwh</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>合计能耗/kwh</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Starttimelbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFTCUM1lbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFTCUM2lbl">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFTCUMP2lbl">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>70</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFTCUMP1lbl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFTCUTC2lbl">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>70</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>－－</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFTCUTC1lbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>－－</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFBCUM1lbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFBCUM2lbl">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFBCUTC2lbl">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>170</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>－－</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFBCUTC1lbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>－－</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFBCUMP1lbl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFBCUMP2lbl">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SIVTC1lbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SIVMP1lbl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>－－</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SIVM2lbl">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SIVTC2lbl">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>220</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SIVMP2bl">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>－－</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SIVM1lbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TotalConsumptionlbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmBtn">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>440</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFBCUTC1lbl_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>－－</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFRMP1lbl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFRM1lbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>再生制动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFRMP2lbl">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFRM2lbl">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VVVFBCUTC2lbl_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>－－</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>VVVFClearBtn</zorder>
  <zorder>SIVClearBtn</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>Starttimelbl</zorder>
  <zorder>VVVFTCUM1lbl</zorder>
  <zorder>VVVFTCUM2lbl</zorder>
  <zorder>VVVFTCUMP2lbl</zorder>
  <zorder>VVVFTCUMP1lbl</zorder>
  <zorder>VVVFTCUTC2lbl</zorder>
  <zorder>VVVFTCUTC1lbl</zorder>
  <zorder>VVVFBCUM1lbl</zorder>
  <zorder>VVVFBCUM2lbl</zorder>
  <zorder>VVVFBCUTC2lbl</zorder>
  <zorder>VVVFBCUTC1lbl</zorder>
  <zorder>VVVFBCUMP1lbl</zorder>
  <zorder>VVVFBCUMP2lbl</zorder>
  <zorder>SIVTC1lbl</zorder>
  <zorder>SIVMP1lbl</zorder>
  <zorder>SIVM2lbl</zorder>
  <zorder>SIVTC2lbl</zorder>
  <zorder>SIVMP2bl</zorder>
  <zorder>SIVM1lbl</zorder>
  <zorder>TotalConsumptionlbl</zorder>
  <zorder>ConfirmBtn</zorder>
  <zorder>VVVFBCUTC1lbl_2</zorder>
  <zorder>VVVFRMP1lbl</zorder>
  <zorder>VVVFRM1lbl</zorder>
  <zorder>label_8</zorder>
  <zorder>VVVFRMP2lbl</zorder>
  <zorder>VVVFRM2lbl</zorder>
  <zorder>VVVFBCUTC2lbl_2</zorder>
  <zorder>returnBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
